woocommerce/woocommerce-android
# Change navigation of the simple payments in the orders

Closes: #7066 

### Description
The PR prepares users to removal of the "simple payments" button in the "Orders" screen

### Testing instructions

* Open orders
* Click on the "Create order" button
* Click on the "simple payments" button
* Notice the new manual on the bottom sheet
* Click that you "got it"
* Notice that the "More" screen has been open

### Images/gif


https://user-images.githubusercontent.com/4923871/183564465-577182bd-367b-4660-badc-ff1e88455539.mp4
